Assets/Demo/DemoGame.cs

```diff
@@ -263,6 +263,7 @@ public void OnStartGameButton()
         Debug.Assert(state == State.InMainMenu);
 
         ////TODO: this should be something that UIActionInputModule can give us without us querying the action directly
+        /*
```

Comment:
So what do we do with this commented out block?

Reply:
Depends a bit on where the demo goes. Likely will get deleted.

Most of the code here has been obsoleted by `PlayerInput`. The functionality commented out is something that `PlayerInputManager` is meant to handle. When the UI stuff gets a pass, IMO makes sense to revisit that too and add the functionality to PlayerInput.